frontend/src/LogInSignUp/LogInSignUp.jsx

```diff
@@ -1,12 +1,60 @@
-import React, { useState } from "react";
+import React, { useContext, useState } from "react";
 import './LogInSignUp.css'
+import {useNavigate,Link} from "react-router-dom"
+// import {UserContext} from "../../UserContext"
+
 
 import user_icon from '../assets/person.png'
 import email_icon from '../assets/email.png'
 import password_icon from '../assets/password.png'
 
 const LogInSignUp = () =>{
     const [action,setAction] = useState('Sign Up');
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const [error, setError] = useState(null)
+    const [loading, setLoading] = useState(false)
+    const {updateUser} = useContext(UserContext)
+    const navigate = useNavigate
+
+    async function handleLogIn(event){
+        event.preventDefault();
+        if(!email || !password){
+            setError("Fill out both fields")
+            return;
+        }
+
+        try{
+            setLoading(true);
+            const response = await fetch("http://localhost:2500/logIn",{
+                method:"POST",
+                headers:{
+                    "Content-Type": "application/json",
+                },
+                body:Json.stringify({
+                    email,
+                    password,
+                }),
+                credentials: 'include'
+            });
+            if(response.ok){
+                const data = await response.json();
```

Comment:
you can also use destructure here inline: `const { user } = await response.json()`